### ISSUE TITLE
Arquivo history.log

### ISSUE THREAD

**Issue by leoheck** [aziesemer/astran#44]
Arquivo de log. Adriel, o arquivo ```history.log``` salva apenas os comandos executados né? Seria interessante colocar nele o resultado de cada comando pra facilitar o debug. O que você acha?

**Comment by aziesemer** [comment]
Não adianta redirecionar a saída. Tipo: Astran > saída.txt

Mas sim, eu tinha a intenção de fazer um gerenciamento de logs melhor mas
nunca saiu.

2015-10-16 15:09 GMT-03:00 Leandro Heck <notifications@github.com>:

**Comment by leoheck** [comment]
Sim, adianta, dá pra fazer muita coisa pelo terminal, mas como o user só
lembra de fazer isso depois de ter executado, ai já ficaria salvo. Os
outros CADs já salvam tudo automaticamente.

--
Leandro Sehnem Heck

2015-10-16 16:03 GMT-03:00 aziesemer <notifications@github.com>:
